Gem push error: java.lang.NoClassDefFoundError: org/slf4j/event/LoggingEvent

Gem repo config:
```yaml
repo:
  type: gem
  storage:
    type: fs
    path: /var/artipie/data
  permissions:
    "*":
      - "*"
```

Gem command:
```txt
GEM_HOST_API_KEY=123 gem push sample-project-1.0.0.gem --host http://localhost:8080/my-gem/
```

Artipie logs:
```txt
$ docker run  -it -v $(pwd)/artipie.yaml:/etc/artipie.yml -v $(pwd):/var/artipie -p 8080:80 artipie-local:latest
[INFO] 2020-07-21 06:24:31,346 main com.artipie.VertxMain - Artipie was started on port 80
[INFO] 2020-07-21 06:24:38,565 vert.x-eventloop-thread-1 com.artipie.http.DockerRoutingSlice - >> POST /my-gem//api/v1/gems HTTP_1_1

Accept-Encoding: gzip;q=1.0,deflate;q=0.6,identity;q=0.3
Accept: */*
User-Agent: Ruby, RubyGems/3.1.2 universal-darwin-19 Ruby/2.6.3 (2019-04-16 patchlevel 62)
Connection: keep-alive
Keep-Alive: 30
Content-Length: 4096
Content-Type: application/octet-stream
Authorization: 123
Host: localhost:8080
[INFO] 2020-07-21 06:24:38,600 FileStorage-1 org.eclipse.jetty.util.log - Logging initialized @7781ms to org.eclipse.jetty.util.log.Slf4jLog
[INFO] 2020-07-21 06:24:38,624 FileStorage-1 com.artipie.SliceFromConfig - Created HTTP client, trustall=false
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.headius.backport9.modules.Modules (file:/usr/lib/artipie.jar) to method sun.nio.ch.NativeThread.signal(long)
WARNING: Please consider reporting this to the maintainers of com.headius.backport9.modules.Modules
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[INFO] 2020-07-21 06:24:42,118 FileStorage-1 com.artipie.http.DockerRoutingSlice - Failure: java.util.concurrent.CompletionException: java.lang.NoClassDefFoundError: org/slf4j/event/LoggingEvent
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyNow(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyStage(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.thenApply(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$MinimalStage.thenApply(Unknown Source)
	at com.artipie.SliceFromConfig.<init>(SliceFromConfig.java:128)
	at com.artipie.http.ArtipieRepositories.lambda$resolve$1(ArtipieRepositories.java:96)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(SingleMap.java:57)
	at io.reactivex.subjects.SingleSubject.onSuccess(SingleSubject.java:141)
	at hu.akarnokd.rxjava2.interop.SingleInterop.lambda$fromFuture$2(SingleInterop.java:82)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.complete(Unknown Source)
	at io.reactivex.internal.observers.ConsumerSingleObserver.onSuccess(ConsumerSingleObserver.java:62)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(SingleMap.java:64)
	at io.reactivex.internal.operators.single.SingleDoOnSuccess$DoOnSuccess.onSuccess(SingleDoOnSuccess.java:60)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(SingleMap.java:64)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(SingleMap.java:64)
	at io.reactivex.internal.operators.flowable.FlowableReduceSeedSingle$ReduceSeedObserver.onComplete(FlowableReduceSeedSingle.java:110)
	at wtf.g4s8.rio.file.ReadSubscriberState.onComplete(ReadSubscriberState.java:76)
	at wtf.g4s8.rio.file.ReadRequest$Next.process(ReadRequest.java:127)
	at wtf.g4s8.rio.file.ReadTaskQueue.run(ReadTaskQueue.java:93)
	at wtf.g4s8.rio.file.CloseChanOnError.run(CloseChanOnError.java:25)
	at wtf.g4s8.rio.file.ErrorOnException.run(ErrorOnException.java:33)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoClassDefFoundError: org/slf4j/event/LoggingEvent
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(java/lang/Class)
	at java.lang.Class.getDeclaredMethods(java/lang/Class)
	at org.jruby.javasupport.binding.MethodGatherer$1.computeValue(org/jruby/javasupport/binding/MethodGatherer.java:256)
	at org.jruby.javasupport.binding.MethodGatherer$1.computeValue(org/jruby/javasupport/binding/MethodGatherer.java:252)
	at java.lang.ClassValue.getFromHashMap(java/lang/ClassValue)
	at java.lang.ClassValue.getFromBackup(java/lang/ClassValue)
	at java.lang.ClassValue.get(java/lang/ClassValue)
	at org.jruby.javasupport.binding.MethodGatherer$2.computeValue(org/jruby/javasupport/binding/MethodGatherer.java:266)
	at org.jruby.javasupport.binding.MethodGatherer$2.computeValue(org/jruby/javasupport/binding/MethodGatherer.java:263)
	at java.lang.ClassValue.getFromHashMap(java/lang/ClassValue)
	at java.lang.ClassValue.getFromBackup(java/lang/ClassValue)
	at java.lang.ClassValue.get(java/lang/ClassValue)
	at org.jruby.javasupport.binding.MethodGatherer.eachAccessibleMethod(org/jruby/javasupport/binding/MethodGatherer.java:176)
	at org.jruby.javasupport.binding.MethodGatherer.getMethods(org/jruby/javasupport/binding/MethodGatherer.java:156)
	at org.jruby.javasupport.binding.MethodGatherer.setupMethods(org/jruby/javasupport/binding/MethodGatherer.java:558)
	at org.jruby.javasupport.binding.MethodGatherer.initialize(org/jruby/javasupport/binding/MethodGatherer.java:127)
	at org.jruby.javasupport.binding.ClassInitializer.initialize(org/jruby/javasupport/binding/ClassInitializer.java:63)
	at org.jruby.javasupport.binding.Initializer.setupProxyClass(org/jruby/javasupport/binding/Initializer.java:54)
	at org.jruby.javasupport.Java.createProxyClass(org/jruby/javasupport/Java.java:549)
	at org.jruby.javasupport.Java.generateClassProxy(org/jruby/javasupport/Java.java:507)
	at org.jruby.javasupport.Java.createProxyClassForClass(org/jruby/javasupport/Java.java:463)
	at org.jruby.javasupport.JavaSupportImpl$3.computeValue(org/jruby/javasupport/JavaSupportImpl.java:127)
	at org.jruby.javasupport.JavaSupportImpl$3.computeValue(org/jruby/javasupport/JavaSupportImpl.java:119)
	at org.jruby.javasupport.JavaSupportImpl.getProxyClassFromCache(org/jruby/javasupport/JavaSupportImpl.java:202)
	at org.jruby.javasupport.Java.getProxyClass(org/jruby/javasupport/Java.java:430)
	at org.jruby.javasupport.Java.getInstance(org/jruby/javasupport/Java.java:357)
	at org.jruby.javasupport.Java.getInstance(org/jruby/javasupport/Java.java:339)
	at org.jruby.javasupport.JavaUtil.convertJavaToUsableRubyObject(org/jruby/javasupport/JavaUtil.java:188)
	at org.jruby.javasupport.JavaMethod.convertReturn(org/jruby/javasupport/JavaMethod.java:518)
	at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(org/jruby/javasupport/JavaMethod.java:457)
	at org.jruby.javasupport.JavaMethod.invokeStaticDirect(org/jruby/javasupport/JavaMethod.java:368)
	at RUBY.<class:SubmitGem>(<script>:28)
	at RUBY.<main>(<script>:9)
	at org.jruby.Ruby.evalScriptlet(org/jruby/Ruby.java:860)
	at org.jruby.Ruby.evalScriptlet(org/jruby/Ruby.java:836)
	at org.jruby.javasupport.JavaEmbedUtils$2.eval(org/jruby/javasupport/JavaEmbedUtils.java:147)
	at com.artipie.gem.GemSlice.rubyLookUp(com/artipie/gem/GemSlice.java:125)
	at com.artipie.gem.GemSlice.<init>(com/artipie/gem/GemSlice.java:86)
	at com.artipie.gem.GemSlice.<init>(com/artipie/gem/GemSlice.java:68)
	at com.artipie.SliceFromConfig.build(com/artipie/SliceFromConfig.java:195)
	at com.artipie.SliceFromConfig.lambda$new$0(com/artipie/SliceFromConfig.java:129)
	at java.util.concurrent.CompletableFuture.uniApplyNow(java/util/concurrent/CompletableFuture)
	at java.util.concurrent.CompletableFuture.uniApplyStage(java/util/concurrent/CompletableFuture)
	at java.util.concurrent.CompletableFuture.thenApply(java/util/concurrent/CompletableFuture)
	at java.util.concurrent.CompletableFuture$MinimalStage.thenApply(java/util/concurrent/CompletableFuture$MinimalStage)
	at com.artipie.SliceFromConfig.<init>(com/artipie/SliceFromConfig.java:128)
	at com.artipie.http.ArtipieRepositories.lambda$resolve$1(com/artipie/http/ArtipieRepositories.java:96)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(io/reactivex/internal/operators/single/SingleMap.java:57)
	at io.reactivex.subjects.SingleSubject.onSuccess(io/reactivex/subjects/SingleSubject.java:141)
	at hu.akarnokd.rxjava2.interop.SingleInterop.lambda$fromFuture$2(hu/akarnokd/rxjava2/interop/SingleInterop.java:82)
	at java.util.concurrent.CompletableFuture.uniWhenComplete(java/util/concurrent/CompletableFuture)
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(java/util/concurrent/CompletableFuture$UniWhenComplete)
	at java.util.concurrent.CompletableFuture.postComplete(java/util/concurrent/CompletableFuture)
	at java.util.concurrent.CompletableFuture.complete(java/util/concurrent/CompletableFuture)
	at io.reactivex.internal.observers.ConsumerSingleObserver.onSuccess(io/reactivex/internal/observers/ConsumerSingleObserver.java:62)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(io/reactivex/internal/operators/single/SingleMap.java:64)
	at io.reactivex.internal.operators.single.SingleDoOnSuccess$DoOnSuccess.onSuccess(io/reactivex/internal/operators/single/SingleDoOnSuccess.java:60)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(io/reactivex/internal/operators/single/SingleMap.java:64)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(io/reactivex/internal/operators/single/SingleMap.java:64)
	at io.reactivex.internal.operators.flowable.FlowableReduceSeedSingle$ReduceSeedObserver.onComplete(io/reactivex/internal/operators/flowable/FlowableReduceSeedSingle.java:110)
	at wtf.g4s8.rio.file.ReadSubscriberState.onComplete(wtf/g4s8/rio/file/ReadSubscriberState.java:76)
	at wtf.g4s8.rio.file.ReadRequest$Next.process(wtf/g4s8/rio/file/ReadRequest.java:127)
	at wtf.g4s8.rio.file.ReadTaskQueue.run(wtf/g4s8/rio/file/ReadTaskQueue.java:93)
	at wtf.g4s8.rio.file.CloseChanOnError.run(wtf/g4s8/rio/file/CloseChanOnError.java:25)
	at wtf.g4s8.rio.file.ErrorOnException.run(wtf/g4s8/rio/file/ErrorOnException.java:33)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(java/util/concurrent/ThreadPoolExecutor)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(java/util/concurrent/ThreadPoolExecutor$Worker)
	at java.lang.Thread.run(java/lang/Thread)
Caused by: java.lang.ClassNotFoundException: org.slf4j.event.LoggingEvent
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredMethods(Unknown Source)
	at java.base/java.lang.Class.getDeclaredMethods(Unknown Source)
	at org.jruby.javasupport.binding.MethodGatherer$1.computeValue(MethodGatherer.java:256)
	at org.jruby.javasupport.binding.MethodGatherer$1.computeValue(MethodGatherer.java:252)
	at java.base/java.lang.ClassValue.getFromHashMap(Unknown Source)
	at java.base/java.lang.ClassValue.getFromBackup(Unknown Source)
	at java.base/java.lang.ClassValue.get(Unknown Source)
	at org.jruby.javasupport.binding.MethodGatherer$2.computeValue(MethodGatherer.java:266)
	at org.jruby.javasupport.binding.MethodGatherer$2.computeValue(MethodGatherer.java:263)
	at java.base/java.lang.ClassValue.getFromHashMap(Unknown Source)
	at java.base/java.lang.ClassValue.getFromBackup(Unknown Source)
	at java.base/java.lang.ClassValue.get(Unknown Source)
	at org.jruby.javasupport.binding.MethodGatherer.eachAccessibleMethod(MethodGatherer.java:176)
	at org.jruby.javasupport.binding.MethodGatherer.getMethods(MethodGatherer.java:156)
	at org.jruby.javasupport.binding.MethodGatherer.setupMethods(MethodGatherer.java:558)
	at org.jruby.javasupport.binding.MethodGatherer.initialize(MethodGatherer.java:127)
	at org.jruby.javasupport.binding.ClassInitializer.initialize(ClassInitializer.java:63)
	at org.jruby.javasupport.binding.Initializer.setupProxyClass(Initializer.java:54)
	at org.jruby.javasupport.Java.createProxyClass(Java.java:549)
	at org.jruby.javasupport.Java.generateClassProxy(Java.java:507)
	at org.jruby.javasupport.Java.createProxyClassForClass(Java.java:463)
	at org.jruby.javasupport.JavaSupportImpl$3.computeValue(JavaSupportImpl.java:127)
	at org.jruby.javasupport.JavaSupportImpl$3.computeValue(JavaSupportImpl.java:119)
	at org.jruby.util.collections.MapBasedClassValue.get(MapBasedClassValue.java:25)
	at org.jruby.javasupport.JavaSupportImpl.getProxyClassFromCache(JavaSupportImpl.java:202)
	at org.jruby.javasupport.Java.getProxyClass(Java.java:430)
	at org.jruby.javasupport.Java.getInstance(Java.java:357)
	at org.jruby.javasupport.Java.getInstance(Java.java:339)
	at org.jruby.javasupport.JavaUtil.convertJavaToUsableRubyObject(JavaUtil.java:188)
	at org.jruby.javasupport.JavaMethod.convertReturn(JavaMethod.java:518)
	at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(JavaMethod.java:457)
	at org.jruby.javasupport.JavaMethod.invokeStaticDirect(JavaMethod.java:368)
	at org.jruby.java.invokers.StaticMethodInvoker.call(StaticMethodInvoker.java:42)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:375)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:174)
	at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:316)
	at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:72)
	at org.jruby.ir.interpreter.InterpreterEngine.interpret(InterpreterEngine.java:80)
	at org.jruby.ir.instructions.DefineClassInstr.INTERPRET_CLASS(DefineClassInstr.java:94)
	at org.jruby.ir.instructions.DefineClassInstr.interpret(DefineClassInstr.java:78)
	at org.jruby.ir.interpreter.StartupInterpreterEngine.processOtherOp(StartupInterpreterEngine.java:178)
	at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:104)
	at org.jruby.ir.interpreter.Interpreter.INTERPRET_ROOT(Interpreter.java:96)
	at org.jruby.ir.interpreter.Interpreter.execute(Interpreter.java:81)
	at org.jruby.ir.interpreter.Interpreter.execute(Interpreter.java:30)
	at org.jruby.ir.IRTranslator.execute(IRTranslator.java:42)
	at org.jruby.Ruby.evalScriptlet(Ruby.java:860)
	at org.jruby.Ruby.evalScriptlet(Ruby.java:836)
	at org.jruby.javasupport.JavaEmbedUtils$2.eval(JavaEmbedUtils.java:147)
	at com.artipie.gem.GemSlice.rubyLookUp(GemSlice.java:125)
	at com.artipie.gem.GemSlice.<init>(GemSlice.java:86)
	at com.artipie.gem.GemSlice.<init>(GemSlice.java:68)
	at com.artipie.SliceFromConfig.build(SliceFromConfig.java:195)
	at com.artipie.SliceFromConfig.lambda$new$0(SliceFromConfig.java:129)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyNow(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyStage(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.thenApply(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$MinimalStage.thenApply(Unknown Source)
	at com.artipie.SliceFromConfig.<init>(SliceFromConfig.java:128)
	at com.artipie.http.ArtipieRepositories.lambda$resolve$1(ArtipieRepositories.java:96)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(SingleMap.java:57)
	at io.reactivex.subjects.SingleSubject.onSuccess(SingleSubject.java:141)
	at hu.akarnokd.rxjava2.interop.SingleInterop.lambda$fromFuture$2(SingleInterop.java:82)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.complete(Unknown Source)
	at io.reactivex.internal.observers.ConsumerSingleObserver.onSuccess(ConsumerSingleObserver.java:62)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(SingleMap.java:64)
	at io.reactivex.internal.operators.single.SingleDoOnSuccess$DoOnSuccess.onSuccess(SingleDoOnSuccess.java:60)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(SingleMap.java:64)
	at io.reactivex.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(SingleMap.java:64)
	at io.reactivex.internal.operators.flowable.FlowableReduceSeedSingle$ReduceSeedObserver.onComplete(FlowableReduceSeedSingle.java:110)
	at wtf.g4s8.rio.file.ReadSubscriberState.onComplete(ReadSubscriberState.java:76)
	at wtf.g4s8.rio.file.ReadRequest$Next.process(ReadRequest.java:127)
	at wtf.g4s8.rio.file.ReadTaskQueue.run(ReadTaskQueue.java:93)
	at wtf.g4s8.rio.file.CloseChanOnError.run(CloseChanOnError.java:25)
	at wtf.g4s8.rio.file.ErrorOnException.run(ErrorOnException.java:33)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worke
```

Unable to push a gem because of org/slf4j/event/LoggingEvent

--- a/pom.xml
+++ b/pom.xml
@@ -201,6 +201,11 @@ SOFTWARE.
       <scope>runtime</scope>
     </dependency>
     <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.8.0-alpha2</version>
+    </dependency>
+    <dependency>
       <groupId>log4j</groupId>
       <artifactId>log4j</artifactId>
       <scope>runtime</scope>

